src/OpenTelemetry.AutoInstrumentation/Util/ActivityHelper.cs

```diff
@@ -70,4 +59,20 @@ internal static void DisposeWithException(this Activity activity, Exception exce
             }
         }
     }
+
+    private static Activity StartActivityWithTags(this ActivitySource activitySource, string operationName, ActivityKind kind, ITags tags)
```

Comment:
Why not make this one public and remove `StartServerActivityWithTags`?

Reply:
I do not have strong opinion about api shape. Fixed in df41937244b183b81854f6758ea7f3f4c00cd2ba